pkg/pb/accounts.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+option go_package = "pb";
```

Comment:
We can keep the protobuf definitions under `satellite` since we don't need other packages directly using them. `pb` package is pretty much the "storj network protocol" rather than "all protobuf definitions".